excellent/functions/functions.go

```diff
@@ -1677,10 +1683,6 @@ func FormatTime(env utils.Environment, args ...types.XValue) types.XValue {
 //
 // @function format_number(number, places [, humanize])
 func FormatNumber(env utils.Environment, args ...types.XValue) types.XValue {
-	if len(args) < 1 || len(args) > 3 {
```

Comment:
unrelated - just noticed don't need this when function is wrapped with `ArgCountCheck`